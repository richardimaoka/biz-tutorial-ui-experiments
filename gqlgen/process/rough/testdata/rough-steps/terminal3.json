{
  "phase": "Installから動かす直前まで",
  "instruction": "go run github.com/99designs/gqlgen generate",
  "commit": "xxxxx",
  "comment": ""
}
