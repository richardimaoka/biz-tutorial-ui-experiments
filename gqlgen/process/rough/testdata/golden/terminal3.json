[
  {
    "step": "",
    "focusColumn": "Terminal",
    "terminalText": "go run github.com/99designs/gqlgen init",
    "terminalType": "command",
    "commit": "d906506ccf6d2e2d0c55680811d0f503b7094d47"
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "commit": "d906506ccf6d2e2d0c55680811d0f503b7094d47",
    "defaultOpenFilePath": "go.mod"
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "go.mod",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "go.sum",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "gqlgen.yml",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "graph/generated.go",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "graph/model/models_gen.go",
    "isFoldFileTree": true
  },
  {
    "step": "",
    "focusColumn": "Source Code",
    "defaultOpenFilePath": "graph/resolver.go",
    "isFoldFileTree": true
  }
]